### ISSUE TITLE
Sharable Potions

### ISSUE THREAD

**Issue by ItsJakObi1** [tiltedphoques/TiltedEvolution#435]
## Description

My friend and I have found it impossible to share crafted potions in anyway. We started the playthrough hoping we could each take on different roles, i.e I am mage that is adept in alchemy and enchanting, he is a warrior adept in smithing etc. however after having grinding through alchemy and gaining the ability to craft helpful potions for us both to use, we couldn't share them.
## Additional context

N/A

**Comment by Merlin6019** [comment]
Have you tried trading the potions using a container?

**Comment by ItsJakObi1** [comment]
> Have you tried trading the potions using a container?

Yeah, tried in playerhomes, in cities, in open sk

**Comment by ItsJakObi1** [comment]
> Have you tried trading the potions using a container?

Yeah we tried in player homes, cities, open Skyrim, dead bodies, even messed about with the potions ingame prid and tried console spawning bud nothing sadly

**Comment by KiraMikamy** [comment]
actually no support for custom potion to be shared. Need someone fix it in the code as the DEV are no longer do that kind of stuff

**Comment by matorina22** [comment]
yeah this is a issue now and enchanted armor as well
